pkg/registry/apis/provisioning/routes.go

```diff
@@ -147,6 +150,12 @@ func (b *APIBuilder) handleSettings(w http.ResponseWriter, r *http.Request) {
 
 	settings := provisioning.RepositoryViewList{
 		Items: make([]provisioning.RepositoryView, len(all)),
+		LegacyStorage: !( // make sure both folders and dashboards are using unified storage
```

Comment:
^^^ NOT great... but we need *something* to tell the frontend that we must export

this seems reasonable enough until we figure out if there is a better place for that